{
  "uid" : "503b0ac0b4199d53",
  "name" : "Home page - smoke test",
  "fullName" : "tests.ui.test_search.TestWeb#test_homepage_title12",
  "historyId" : "b2e57dac307fb5f5a232440dbae3faf9",
  "time" : {
    "start" : 1649270049308,
    "stop" : 1649270050205,
    "duration" : 897
  },
  "status" : "passed",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "browser",
    "time" : {
      "start" : 1649270048530,
      "stop" : 1649270048530,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1649270043746,
      "stop" : 1649270043831,
      "duration" : 85
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "web_setup",
    "time" : {
      "start" : 1649270048530,
      "stop" : 1649270049308,
      "duration" : 778
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "web_setup::0",
    "time" : {
      "start" : 1649270050206,
      "stop" : 1649270050334,
      "duration" : 128
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "tag",
    "value" : "@pytest.mark.usefixtures('web_setup')"
  }, {
    "name" : "parentSuite",
    "value" : "tests.ui"
  }, {
    "name" : "suite",
    "value" : "test_search"
  }, {
    "name" : "subSuite",
    "value" : "TestWeb"
  }, {
    "name" : "host",
    "value" : "fv-az201-861"
  }, {
    "name" : "thread",
    "value" : "3495-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.ui.test_search"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 3,
        "skipped" : 0,
        "passed" : 1,
        "unknown" : 0,
        "total" : 4
      },
      "items" : [ {
        "uid" : "3c03ebe525b29cad",
        "reportUrl" : "https://rupu1906.github.io/docker-selenium/41//#testresult/3c03ebe525b29cad",
        "status" : "broken",
        "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f43aff65c00>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1649193829522,
          "stop" : 1649193829522,
          "duration" : 0
        }
      }, {
        "uid" : "1d7beeccc4082119",
        "reportUrl" : "https://rupu1906.github.io/docker-selenium/39//#testresult/1d7beeccc4082119",
        "status" : "broken",
        "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f2a6a6b1b70>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1649193204415,
          "stop" : 1649193204415,
          "duration" : 0
        }
      }, {
        "uid" : "1f2fafe0e93dfd13",
        "reportUrl" : "https://rupu1906.github.io/docker-selenium/38//#testresult/1f2fafe0e93dfd13",
        "status" : "broken",
        "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f81bfa61c00>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1649183948831,
          "stop" : 1649183948831,
          "duration" : 0
        }
      } ]
    },
    "tags" : [ "@pytest.mark.usefixtures('web_setup')" ]
  },
  "source" : "503b0ac0b4199d53.json",
  "parameterValues" : [ ]
}